pybamm/solvers/c_solvers/idaklu/CasadiSolverOpenMP.cpp

```diff
@@ -516,3 +647,11 @@ Solution CasadiSolverOpenMP::solve(
 
   return sol;
 }
+
+void CasadiSolverOpenMP::CheckErrors(int& flag) {
```

Comment:
This can be made const since it never changes
```suggestion
void CasadiSolverOpenMP::CheckErrors(int const & flag) {
```